cabal-install/Distribution/Client/ProjectPlanning.hs

```diff
@@ -1,4 +1,4 @@
-{-# LANGUAGE CPP, RecordWildCards, NamedFieldPuns, RankNTypes #-}
+{-# LANGUAGE RecordWildCards, NamedFieldPuns, RankNTypes #-}
 {-# LANGUAGE ViewPatterns #-}
```

Comment:
Given that e.g. ViewPatterns is now listed in other-extensions, shouldn't these also be removed? Or am I missing something?